src/PairedUrlStructure/LegacyTransitionalUrlStructure.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Class LegacyTransitionalUrlStructure.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\PairedUrlStructure;
+
+use AmpProject\AmpWP\PairedUrlStructure;
+
+/**
+ * Descriptor for paired URL structures that end in /amp/ path suffix for non-hierarchical posts and ?amp for others.
```

Comment:
This doesn't look right base don the code.

Reply:
Correct. I copied it from `LegacyReaderUrlStructure`. Copy pasta fixed in e32401749.